storage/src/persistent/commit_log.rs

```diff
@@ -308,4 +290,23 @@ mod tests {
             ranges
         );
     }
+
+    #[test]
+    fn test_commit_log_storage() {
+        let path = env!("LOG_DIR");
```

Comment:
@mambisi 
better change it here something like:
```
    let out_dir = env::var("OUT_DIR").expect("OUT_DIR is not defined");
    Path::new(out_dir.as_str()).join("LOG_DIR")
```

or what is LOG_DIR anyway?

Reply:
It was for debugging purposes, will be removed